.github/workflows/buildx-image.yml

```diff
@@ -0,0 +1,56 @@
+name: buildx-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      source-tag:
+        description: 'BuildKit source Docker tag'
+        required: true
+        default: 'latest'
+      dest-tag:
+        description: 'Default BuildKit Docker tag for buildx'
+        required: true
+        default: 'buildx-stable-1'
+      dry-run:
+        description: 'Dry run'
+        required: false
+        default: 'true'
+
+env:
+  REPO_SLUG_TARGET: "moby/buildkit"
+
+jobs:
+  create:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        flavor:
+          - ''
+          - '-rootless'
+    steps:
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to DockerHub
+        if: inputs.dry-run != 'true'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Create
+        run: |
+          DRYRUN_FLAG=""
+          if [ "${{ inputs.dry-run }}" = "true" ]; then
+            DRYRUN_FLAG="--dry-run"
+          fi
+          docker buildx imagetools create ${DRYRUN_FLAG} --tag \
+            "${{ env.REPO_SLUG_TARGET }}:${{ inputs.dest-tag }}${{ matrix.flavor }}" \
+            "${{ env.REPO_SLUG_TARGET }}:${{ inputs.source-tag }}${{ matrix.flavor }}"
```

Comment:
still more complex needed. rootless image is at `:rootless` not `:latest-rootless`. The current would work if it validates that only `vX.X.X` tags are allowed (and not the current default value).

Reply:
That's right. I've updated the workflow and also added a comment about the potential outcome.